ruby/rails_rubocop.yml

```diff
@@ -0,0 +1,20 @@
+inherit_from: base_rubocop.yml
+
+AllCops:
+  DisplayCopNames: true
```

Comment:
- [ ] Jag tycker nog att `DisplayCopNames` borde ligga i basen.
- [ ] Du har inherit from base_rubocop.yml, men filen heter rubocop.yml
- [ ] Förresten borde vi kanske lägga in `DisplayStyleGuide` också
```
AllCops:
  DisplayCopNames: true
  DisplayStyleGuide: true
```
Så ser man varför också, det är mycket hjälpsamt.